graveler/graveler.go

```diff
@@ -1327,3 +1372,36 @@ func (g *Graveler) getCommitsForMerge(ctx context.Context, repositoryID Reposito
 	}
 	return fromCommit, toCommit, baseCommit, nil
 }
+
+func newErrAbortedByHook(err error) error {
+	if err == nil {
+		return ErrAbortedByHook
+	}
+	merr := multierror.Append(ErrAbortedByHook, err)
+	merr.ErrorFormat = func(errs []error) string {
+		return fmt.Sprintf("%s: %s", errs[0], errs[1])
```

Comment:
This works for almost everything, unless original error was of type `*multierror.Error`